frontend/comp/Uploader.jsx

```diff
@@ -2,14 +2,23 @@ import React from 'react';
 import * as blockstack from 'blockstack'
 import md5 from 'md5'
 
-export default class Uploader  extends React.Component {
+export default class Uploader extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isButtonDisabled: false
+    }
+  }
 
   upload (evt) {
     evt.preventDefault();
 
     let input = this.input
     let file = input.files[0]
 
+    this.setState({isButtonDisabled: true})
+    this.props.updateIsLoading(true)
```

Comment:
Here we have change two states for the same thing. There is no need for this duplication.
Let's just change the state in the index and put it as property inside here, to react on.

Reply:
ok good idea, I changed it